plugins/meta/flannel/flannel.go

```diff
@@ -241,6 +248,119 @@ func cmdAdd(args *skel.CmdArgs) error {
 	return delegateAdd(args.ContainerID, n.DataDir, n.Delegate)
 }
 
+func cmdAddWindows(containerID string, n *NetConf, fenv *subnetEnv) error {
+	if n.WindowsDelegate == nil {
+		n.WindowsDelegate = make(map[string]interface{})
+	} else {
```

Comment:
these checks appear identical to those in the other `cmdAdd` function, with only a difference in the error string.  Could you extract a function with the common code?

Reply:
Since I'm removing the "WindowsDelegate" attribute, I'll just move the call to cmdAddWindows down after the checks and remove the dups.